src/modules/sched-simple/sched.c

```diff
@@ -208,6 +225,28 @@ static int try_alloc (flux_t *h, struct simple_sched *ss)
     return rc;
 }
 
+static void annotate_reason_pending (struct simple_sched *ss)
+{
+    int jobs_ahead = 0;
+
+    if (flux_module_debug_test (ss->h, DEBUG_ANNOTATE_REASON_PENDING, false))
```

Comment:
Ah, this could be the problem with the impacted throughput, shouldn't this be `if (!flux_module_debug_test (...)`?

Reply:
doh! yup, and in combo with the bitmask mistake above, that's why tests were passing.